packages/story-editor/src/elements/video/captions.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { ResourcePropTypes } from '@googleforcreators/media';
+
+function Captions({ tracks, defaultKind = 'metadata' }) {
+  if (!tracks) {
+    return null;
+  }
+
+  return tracks.map(({ srclang, label, kind, track, id: key }, i) => (
+    <track
+      srcLang={srclang}
+      label={label}
+      kind={kind || defaultKind}
```

Comment:
```suggestion
      // Hides the track from the user
      // Displaying happens in MediaCaptionsLayer instead.
      kind={kind || defaultKind}
```

Reply:
That comment should be in `display.js`, not here

Reply:
Comment moved.